### PR TITLE
Multiple Redirection URL

### PR DESCRIPTION
**Pull request by SanjuPal01**
Signed-off-by: SanjuPal01 <sanju.sanju@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
Added the functionality to patch multiple redirection URL for Dex client from Automate Config.

### :chains: Related Resources
https://chefio.atlassian.net/browse/KINETICS-175

### :athletic_shoe: How to Build and Test the Change
```
rebuild components/automate-cli
rebuild components/automate-deployment
rebuild components/automate-gateway
rebuild components/automate-dex
chef-automate config patch dev/redirection_url.toml
chef-automate config show
```

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: api/config/dex/config_request.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ func NewConfigRequest() *ConfigRequest {
 				Disclosure:           &ConfigRequest_V1_Disclosure{},
 				Banner:               &ConfigRequest_V1_Banner{},
 				InvalidLoginAttempts: &ConfigRequest_V1_InvalidLoginAttempts{},
+				Redirection:          &ConfigRequest_V1_Redirection{},
```
Comment (Dmaddu) [inline api/config/dex/config_request.go:33]: Rename Redirection to RedirectionURIs in all the files.

**Review by Dmaddu — CHANGES_REQUESTED** [review]

File: api/config/dex/config_request.go (hunk 1 above)
Comment (SanjuPal01) [reply api/config/dex/config_request.go:33]: okay

File: api/config/dex/config_request.go [hunk 2]
```diff
@@ -30,6 +30,7 @@ func NewConfigRequest() *ConfigRequest {
 				Disclosure:           &ConfigRequest_V1_Disclosure{},
 				Banner:               &ConfigRequest_V1_Banner{},
 				InvalidLoginAttempts: &ConfigRequest_V1_InvalidLoginAttempts{},
+				RedirectionUris:      &ConfigRequest_V1_RedirectionURIs{},
```
Comment (Dmaddu) [inline api/config/dex/config_request.go:33]: This breaks the new installation, we need to have /signin as the default value in the list.
please make the changes in the following function. 
func DefaultConfigRequest() *ConfigRequest {

**Review by Dmaddu — CHANGES_REQUESTED** [review]

File: api/config/dex/config_request.go (hunk 2 above)
Comment (SanjuPal01) [reply api/config/dex/config_request.go:33]: Okay

**Pull request merged**